org.adempiere.install/src/org/compiere/install/Setup.java

```diff
@@ -151,17 +149,18 @@ else if (e.getSource() == menuHelpInfo)
 	 */
 	public static void main(String[] args)
 	{
-		CLogMgt.initialize(true);
-		Handler fileHandler = new CLogFile(System.getProperty("user.dir"), false, false);
-		CLogMgt.addHandler(fileHandler);
+		CLogMgt.initialize(false);
+		// Duplicate log file - not required
+		// Handler fileHandler = new CLogFile(System.getProperty("user.dir"), false, false);
+		// CLogMgt.addHandler(fileHandler);
 		//	Log Level
 		if (args.length > 0)
 			CLogMgt.setLevel(args[0]);

```

Comment:
I think we should validate the arguments here.

Reply:
This was legacy code - but is a good idea to check it - I added the check to the three setup programs